action/protocol/staking/candidate_center.go

```diff
@@ -440,14 +444,20 @@ func (cb *candBase) all() CandidateList {
 	return list
 }
 
-func (cb *candBase) commit(change *candChange) (int, error) {
+func (cb *candBase) commit(change *candChange, keepAliasBug bool) (int, error) {
```

Comment:
if we could have the flag in the `candBase`, then I think it's inefficient to pass it down every time